app/backend/src/couchers/servicers/api.py

```diff
@@ -492,6 +511,8 @@ def RespondFriendRequest(self, request, context):
         with session_scope() as session:
             friend_request = (
                 session.query(FriendRelationship)
+                .join(User, FriendRelationship.from_user_id == User.id)
```

Comment:
I like to keep the expression as `Foreign.column == This.column` for consistency, `Foreign` is `User` here and `This` is `FriendRelationship`